src/helpers/uuid.ts

```diff
@@ -3,20 +3,9 @@
  * */
 
 export class UuidGenerator {
-  private isFastIdStrategy = false;
-
-  private fastIdStart = 0;
-
-  setIsFastStrategy(isFast: boolean) {
-    this.isFastIdStrategy = isFast;
-  }
-
   uuidv4(): string {
-    if (this.isFastIdStrategy) {
-      this.fastIdStart++;
-      return String(this.fastIdStart);
-      //@ts-ignore
-    } else if (window.crypto && window.crypto.getRandomValues) {
+    //@ts-ignore
```

Comment:
I think this ts-ignore is useless

Reply:
very much useful becasue ts won't let you sum an array of numbers with a number, even though of course it works in real life conditions ^^

Reply:
Was taI was talking about the ts-ignore on `if (window.crypto && window.crypto.getRandomValues)` :upside_down_face: